_js/build

```diff
@@ -1,12 +1,13 @@
 #!/bin/sh
 
-if ! go version | grep -q go1.12; then
+if ! command -v "go1.12" >/dev/null 2>&1; then
 	echo "Go 1.12.x required to build."
+	echo "run \`go get golang.org/dl/go1.12 && go1.12 download\` first"
```

Comment:
undo this as well. you can set up your PATH accordingly instead, like:

```
mkdir -p some/bin
ln -sf $(which go1.12) some/bin/go
export PATH="$PWD/some/bin:$PATH"
# run your command here
```

Most people will likely use the 1.12 wrapper, but we shouldn't add a hard dependency on it.

Reply:
The main go package `Requires Go 1.14 or later`, so I imagined that we have to install multiple go versions, but if you don't like it, I can revert it.

Reply:
Yes, you will need to install multiple versions unfortunately, but what I'm saying is that you can do that outside the script pretty easily without having to hard-code it into the script :)

Reply:
It seems redundant to change go versions between folders at a same project personally, but of course it's fine to keep it as-is.